packages/ragbits-core/src/ragbits/core/prompt/lab/app.py

```diff
@@ -112,7 +112,7 @@ def send_prompt_to_llm(state: PromptState) -> str:
     except Exception as e:  # pylint: disable=broad-except
         response = str(e)
 
-    return response
+    return response  # type: ignore
```

Comment:
Why? It seems to me that this will cause the app to crash for any streaming prompt - the problem that `# type: ignore` silences here is really a concern - it will cause `send_prompt_to_llm` to return a type different than what it declares and then it will cause gradio to try to set this non-string return value as a text content for a textbox.

Reply:
resolved by implementing separate method for streaming